src/Services/SwaggerService.php

```diff
@@ -180,6 +178,9 @@ public function addData(Request $request, $response)
         $this->parseRequest();
         $this->parseResponse($response);
 
+        $this->limitResponseData();
+        $this->cutExceptions();
```

Comment:
please cut data on the parsing response step, not after the data new data already added